zrml/prediction-markets/src/lib.rs

```diff
@@ -131,24 +121,9 @@ mod pallet {
                 slash_market_creator(T::OracleBond::get());
             }
 
-            // Delete market's outcome assets, clear market and delete pool if necessary.
-            let mut destroy_asset = |asset: Asset<_>| -> Option<usize> {
-                if let Ok((total_accounts, accounts)) =
-                    T::AssetManager::accounts_by_currency_id(asset)
-                {
-                    share_accounts = share_accounts.saturating_add(accounts.len());
-                    let _ = T::AssetManager::destroy_all(asset, accounts.iter().cloned());
-                    Some(total_accounts)
-                } else {
-                    // native currency case
-                    None
-                }
-            };
-            for asset in outcome_assets.into_iter() {
-                if let Some(total) = destroy_asset(asset) {
-                    total_accounts = total;
-                }
-            }
+            //NOTE: Currently we don't clean up outcome assets.
+            //TODO: Above needs more work on https://github.com/zeitgeistpm/zeitgeist/issues/101
```

Comment:
I don't know, why the CI doesn't complain with this TODO comment, but the style is normally the following.

```suggestion
            // TODO(#792): Remove outcome assets for accounts! Delete asset of `orml_tokens` with storage migration.
```

Reply:
Maybe reference this PR with the full link to it in this comment to, that we later have an idea, what code was deleted and can be probably reused https://github.com/zeitgeistpm/zeitgeist/pull/806

Maybe add  this comment also to on_idle or on_initialize.